state/src/impls/shard_level.rs

```diff
@@ -714,6 +713,7 @@ impl<'db> ShardLevelState<'db> {
         self.cache.remove_asset(account);
     }
 
+    #[allow(dead_code)]
     fn kill_asset_scheme(&mut self, account: &AssetSchemeAddress) {
```

Comment:
I think it's fine to remove this method.

Reply:
I'll remove it.

Reply:
Removed.